### PR TITLE
Feature/adp

### PR DESCRIPTION
**Pull request by gonzalo-munillag**
## Description
I added floordiv, lt, le, ge, eq, ne

## Affected Dependencies
none

## How has this been tested?
Minor additions, following the same template as the rest of the code. No test needed.

## Checklist
- [ x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [ x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ x] My changes are covered by tests

### CODE REVIEW

**Comment by tudorcebere** [comment]
Adding this at a later moment with tests as well. Closing for now.

File: packages/syft/src/syft/core/tensor/autodp/single_entity_phi.py [hunk 1]
```diff
@@ -120,7 +120,212 @@ def __gt__(self, other):
...
+                entity=entity,
+                min_vals=min_vals,
+                max_vals=max_vals,
+                scalar_manager=self.scalar_manager,
+            )
+
+        else:
+            return NotImplemented
+    
+        def __le__(self, other):
```
Comment (replomancer) [inline packages/syft/src/syft/core/tensor/autodp/single_entity_phi.py:206]: This line needs to be unindented.

File: packages/syft/src/syft/core/tensor/autodp/single_entity_phi.py [hunk 2]
```diff
@@ -120,7 +120,212 @@ def __gt__(self, other):
...
+                entity=entity,
+                min_vals=min_vals,
+                max_vals=max_vals,
+                scalar_manager=self.scalar_manager,
+            )
+
+        else:
+            return NotImplemented
+    
+        def __eq__(self, other):
```
Comment (replomancer) [inline packages/syft/src/syft/core/tensor/autodp/single_entity_phi.py:247]: Indentation.

File: packages/syft/src/syft/core/tensor/autodp/single_entity_phi.py [hunk 3]
```diff
@@ -120,7 +120,212 @@ def __gt__(self, other):
...
+                entity=entity,
+                min_vals=min_vals,
+                max_vals=max_vals,
+                scalar_manager=self.scalar_manager,
+            )
+
+        else:
+            return NotImplemented
+    
+        def __ne__(self, other):
```
Comment (replomancer) [inline packages/syft/src/syft/core/tensor/autodp/single_entity_phi.py:288]: Also here.

**Review by replomancer — COMMENTED** [review]
Good stuff. I haven't run it, but I can see three IndentationErrors.

**Pull request closed without merging**